README-JP.md

```diff
@@ -0,0 +1,139 @@
+
+## インストール
+
+`go get`を使用してSDKをインストールします：
+
+```sh
+go get -u github.com/aliyun/alibaba-cloud-sdk-go/sdk
+```
+
+## クイックスタート
+
+始める前に、Alibaba Cloudアカウントにサインアップし、[認証情報](https://usercenter.console.aliyun.com/#/manage/ak)を取得する必要があります。
+
+### クライアントの作成
+
+```go
+package main
+
+import "github.com/aliyun/alibaba-cloud-sdk-go/sdk"
+
+func main() {
+  client, err := sdk.NewClientWithAccessKey("REGION_ID", "ACCESS_KEY_ID", "ACCESS_KEY_SECRET")
+  if err != nil {
+    // 例外処理
+    panic(err)
+  }
+}
+```
+
+### ROAリクエスト
+
+```go
+package main
+
+import "github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+
+func main() {
+  request := requests.NewCommonRequest()        // 共通リクエストを作成
+  request.Method = "GET"                        // リクエストメソッドを設定
+  request.Product = "CS"                        // 製品を指定
+  request.Domain = "cs.aliyuncs.com"            // ホストを指定すると、ロケーションサービスは有効になりません。たとえば、認証タイプがBearer Tokenのサービスは指定する必要があります
+  request.Version = "2015-12-15"                // 製品バージョンを指定
+  request.PathPattern = "/clusters/[ClusterId]" // ROAスタイルのパスルールを指定
+  request.Scheme = "https"                      // リクエストスキームを設定。デフォルトはhttp
+  request.ApiName = "DescribeCluster"           // 製品インターフェースを指定
+  request.QueryParams["ClusterId"] = "123456"   // パス内のパラメータに値を割り当て
+  request.QueryParams["RegionId"] = "region_id" // リクエストされたregionIdを指定。指定しない場合は、クライアントのregionId、次にデフォルトのregionIdを使用
+  request.TransToAcsRequest()                   // 共通リクエストをacsリクエストに変換。これはクライアントによって使用されます。
+}
+```
+
+### RPCリクエスト
+
+```go
+package main
+
+import "github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+
+func main() {
+  request := requests.NewCommonRequest()                // 共通リクエストを作成
+  request.Method = "POST"                               // リクエストメソッドを設定
+  request.Product = "Ecs"                               // 製品を指定
+  request.Domain = "ecs.aliyuncs.com"                   // ホストを指定すると、ロケーションサービスは有効になりません。たとえば、認証タイプがBearer Tokenのサービスは指定する必要があります
+  request.Version = "2014-05-26"                        // 製品バージョンを指定
+  request.Scheme = "https"                              // リクエストスキームを設定。デフォルトはhttp
+  request.ApiName = "CreateInstance"                    // 製品インターフェースを指定
+  request.QueryParams["InstanceType"] = "ecs.g5.large"  // パス内のパラメータに値を割り当て
+  request.QueryParams["RegionId"] = "region_id"         // リクエストされたregionIdを指定。指定しない場合は、クライアントのregionId、次にデフォルトのregionIdを使用
+  request.TransToAcsRequest()                           // 共通リクエストをacsリクエストに変換。これはクライアントによって使用されます。
+}
+```
+
+## ドキュメント
+
+- [Requirements](docs/0-Requirements-JP.md)
+- [Installation](docs/1-Installation-JP.md)
+- [Client & Credentials](docs/2-Client-JP.md)
+- [SSL Verify](docs/3-Verify-JP.md)
+- [Proxy](docs/4-Proxy-JP.md)
+- [Timeout](docs/5-Timeout-JP.md)
+- [Debug](docs/6-Debug-JP.md)
+- [Logger](docs/7-Logger-JP.md)
+- [Concurrent](docs/8-Concurrent-JP.md)
+- [Asynchronous Call](docs/9-Asynchronous-JP.md)
+- [Package Management](docs/10-Package-Management-JP.md)
+- [Endpoint](docs/11-Endpoint-JP.md)
```

Comment:
The above documents are not yet prepared. Can you link to the corresponding English documents first?